docs/Users_Guide/getting_started.rst

```diff
@@ -0,0 +1,356 @@
+   
+* {MET TOOL}_CLIMO_CDF
+   
+This can be found in Grid-Stat, Point-Stat, Gen-Ens-Prod, and Ensemble-Stat
+tools.
+
+What is the geographic location of the model data being evaluated? Are there specific areas of interest for the evaluation?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Masking regions are what METplus uses to define verification areas of
+interest. These can be defined prior to running tools using the
+Gen-Vx-Mask tool, or during run-time using the METPLUS_MASK_DICT options.
+
+What domain should be used to evaluate on the model domain, observation domain (if gridded), or some other domain?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The decision to evaluate on model or observation/analysis domain is
+user-specific but the user may want to consider the following:
+
+* Regridding to the courser domain will smooth high resolution information
+  that may be important but smoother forecasts tend to score better.
+   
+* Regridding to a finer domain essentially adds in additional information
+  that is not real.
+   
+* One way to avoid the interpolation debate is to regrid both to a third
+  grid.
+   
+Regridding in METplus can be completed using the Regrid-Data-Plane tool if
+the fields will be used more than once.
+
+Regridding can also be done on the fly using the {Tool}_REGRID_TO_GRID.
+All grid-to-grid verification tools have the regridding capability in it.
+
+What is the evaluation time period? Retrospective with specific dates? Ongoing, near-real-time evaluation? Or both retrospective and realtime?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Basically, running retrospectively means that the observations/analyses are
+already available on disk and running in realtime is when the system needs
+to wait for the observations to be available on the system.
+
+In METplus, the LOOP_BY configuration can be used.
+
+LOOP_BY = VALID or REALTIME to have METplus proceed through the data based
+on Valid Time.
+
+LOOP_BY = INIT or RETRO to have METplus proceed through the data based
+on Initialization Time.
+
+How should the testing and evaluation project be broken down into METplus
+Use Cases? One large one or multiple smaller ones?
+
+* How will METplus be run? Manually? Scheduled, through cron?
+  Automated via a workflow manger (e.g. Rocoto, EC-Flow, Rose-Cylc)?
+
+  * If run manually, this can be done.
+    
+  * If scheduled through cron, a bash or csh script can be written to
+    set up environment variables to pass into METplus.
+    
+  * If automated via a workflow manager, it is recommended the user consider
+    configuring the use cases to run smaller amounts of data.
+    
+* Where will METplus be run? Local machine, project machine, HPC system,
```

Comment:
This also seems like it should be its own header.

Reply:
Done.